packages/turf-center-mean/index.js

```diff
@@ -39,14 +39,15 @@ function centerMean(geojson, options) {
     var sumYs = 0;
     var sumNs = 0;
     geomEach(geojson, function (geom, featureIndex, properties) {
-        var weightValue = properties[weight] || 1;
-        if (!isNumber(weightValue)) throw new Error('weight value must be a number for feature index ' + featureIndex);
-        weightValue = Number(weightValue);
-        if (weightValue > 0) {
+        var weightValue = properties[weightTerm];
```

Comment:
You don't need to define both `weight` & `weightValue`, changing `weightValue` to Number won't mutate the original input, so you will be ok to do this:

```js
var weight = properties[weightTerm];
weight = (weight === undefined || weight === null) ? 1 : weight;
```
The only reason why you would need to have both `weight` & `weightValue` is if you are comparing both of those values down the road (which isn't the case).